### PR TITLE
Helm/practice meiga

### PR DESCRIPTION
**Pull request by AlbertoFemenias**
- "Practica guiada Helm-Meiga" is in a state where it could be published and it will be useful. 

- There is still some content that needs to be added at the end relating **helm-unit-tests**. Right now Juan Deus is working on this point and this information will be added as soon as the work is complete.  #42

### CODE REVIEW

**Review by pablosanchezpaz — APPROVED** [review]

**Review by frmadem — COMMENTED** [review]
@AlbertoFemenias on the beginning of the introduction, you should link to the original task on the docker course.

File: cursos/helm/02_helm_charts/02_practica_guiada_meiga.md [hunk 1]
```diff
@@ -92,10 +131,86 @@ Y cambiamos la línea correnpondiente de `frontend-service.yaml`
...
+### d) Contenido del secrets
+Vamos a exponer la contraseña de nuestra base de datos para que se pueda modificar desde `values.yaml`:
+```yaml
+#values.yaml
+...
+secretos:
+  rootpass: contrasinal
+```
+
+Vamos a utilizar una de las funciones que nos proporciona [Sprig](http://masterminds.github.io/sprig/) para introducir como texto nuestra contraseña en `values.yaml` y que Helm la inyecte dentro de `secret.yaml` en formato base64:
```
Comment (frmadem) [inline cursos/helm/02_helm_charts/02_practica_guiada_meiga.md:204]: @AlbertoFemenias We should create a section for introduce the pipes and the function system of Helm's sprig. Should it be a new issue/task?

File: cursos/helm/02_helm_charts/02_practica_guiada_meiga.md [hunk 2]
```diff
@@ -135,4 +251,3 @@ $ helm install mimeiga meiga-project/ -f misvalores.yaml
 Si tienes algún problema con esta práctica guiada puedes ver el proyecto solución completo [aquí](https://github.com/prefapp/formacion/tree/master/cursos/helm/codigo_practica_guiada_meiga/meiga-helm).
```
Comment (frmadem) [inline cursos/helm/02_helm_charts/02_practica_guiada_meiga.md:251]: @AlbertoFemenias I think you should mention the helm template option.

File: cursos/helm/02_helm_charts/02_practica_guiada_meiga.md (hunk 1 above)
Comment (AlbertoFemenias) [reply cursos/helm/02_helm_charts/02_practica_guiada_meiga.md:204]: I added yesterday issue #44 with some concepts I thought should be included, I will add "pipes and functions in Helm"

File: cursos/helm/02_helm_charts/02_practica_guiada_meiga.md (hunk 2 above)
Comment (AlbertoFemenias) [reply cursos/helm/02_helm_charts/02_practica_guiada_meiga.md:251]: Will do

**Pull request merged**